x-pack/plugins/observability/common/locators/paths.ts

```diff
@@ -13,6 +13,7 @@ export const ALERTS_PATH = '/alerts' as const;
 export const ALERT_DETAIL_PATH = '/alerts/:alertId' as const;
 export const EXPLORATORY_VIEW_PATH = '/exploratory-view' as const; // has been moved to its own app. Keeping around for redirecting purposes.
 export const RULES_PATH = '/alerts/rules' as const;
+export const RULES_LOGS_PATH = '/alerts/rules/logs' as const;
```

Comment:
Do we need to add a new path, can't we have a query parameter instead for the selected tab?

The reason that I ask is related to breadcrumb, I wonder if it needs to match the path that we are on or not. Also, the page is technically the same, we are just changing a tab. I don't have a strong opinion about it, just was curious about the reasoning.